tests/Feature/CustomerRegistrationTest.php

```diff
@@ -0,0 +1,102 @@
+<?php
+
+namespace Tests\Feature;
+
+use App\Models\User;
+use App\Classes\Facades\CustomerRegistrationOptions;
+use Illuminate\Foundation\Testing\RefreshDatabase;
+use Illuminate\Foundation\Testing\WithFaker;
+use Illuminate\Support\Carbon;
+use Tests\TestCase;
+
+class CustomerRegistrationTest extends TestCase
+{
+    use RefreshDatabase;
+
+
+    public function test_redirect_to_customers_registration(){
```

Comment:
you're not testing the redirect you are testing the views. I think this would be better if we have the function name to be like "test_customers_registration_view"